reimburse.py

```diff
@@ -93,14 +94,18 @@ def compute_all_reimburse(event, context, user):
     db.authenticate(config.DB_USER,config.DB_PASS)
     tests = db[config.DB_COLLECTIONS['users']]
 
-    reg_stat_to_money = {i['_id']: i['amount'] for i in tests.aggregate([{"$group":{"_id": "$registration_status", "amount": {"$sum": "$travelling_from.reimbursement"}}}])}
-    print(reg_stat_to_money)
-    given = reg_stat_to_money.get('coming', 0) + reg_stat_to_money.get('confirmed', 0)
-    config.TRAVEL.BUDGET -= given
-    if config.TRAVEL.BUDGET <= 0:
-        return {'statusCode': 512, 'body': "budget allocated fully!"}
-
-    users = list(tests.find({"travelling_from": {"$exists": True}, "travelling_from.addr_ready": True, "registration_status": "registered"}))
+    if 'day-of' not in event or not event['day-of']:
+        reg_stat_to_money = {i['_id']: i['amount'] for i in tests.aggregate([{"$group":{"_id": "$registration_status", "amount": {"$sum": "$travelling_from.reimbursement"}}}])}
+        print(reg_stat_to_money)
+        given = reg_stat_to_money.get('coming', 0) + reg_stat_to_money.get('confirmed', 0)
+        config.TRAVEL.BUDGET -= given
```

Comment:
this irks me out a bit because your a doing a mutation operation on the config object. it may work out that because lambdas reload the module before execution that this doesn't matter, but really config should stay constant in memory. if this is not the intended affect then this mutable state should live in the database.

Reply:
Yeah, that was the intended effect - that it'd reset. It's a messy way to do it, but I didn't know what semantics were going to be, so avoided thinking about it too much. I'll make it local variable that gets passed as a param.

We don't need the remaining budget in a DB I think, because it's not too hard to query for. If you disagree, we can add something to the misc bucket later on.